Patch104pZH/GameFilesEdited/Data/INI/Object/ChemicalGeneral.ini

```
@@ -5693,6 +5693,10 @@ Object Chem_GLAScudStorm
     DeathFX       = FX_StructureMediumDeath
   End
 
+  Behavior  = CreateObjectDie ModuleTag_20
```

```suggestion
  Behavior  = CreateObjectDie ModuleTag_22
```
To keep this number consistent between all variants.